distribution/docs/src/main/resources/_contents/_resources/content-directory-monitor-contents.adoc

```diff
@@ -47,8 +47,16 @@ Regardless of the option, the ${branding} takes each file in a monitored directo
 ** If the original file is modified, the metacard is updated to reflect the new content.
 ** If the original file is renamed, the old metacard is deleted and a new metacard is created.
 
-.Atribute Mapper
-The Content Directory Monitor supports setting metacard attributes directly when ${branding} ingests a file. Custom mappings are entered in the form:
+.Parallel Processing
+The CDM supports parallel processing of files (up to 8 files processed concurrently).  This is configured by setting the number of *Maximum Concurrent Files* in the configuration.  A maximum of 8 is imposed to protect system resources.
+
+.Read Lock
```

Comment:
Is it worth noting that this does not apply to the `monitor in place` processing directive?